tests/test_postcard.py

```diff
@@ -96,10 +96,10 @@ def test_create_postcard_inline(self):
                 'address_state': 'CA'
             },
             front = '<h1>{{front_name}}</h1>',
+            back = '<h1>{{back_name}}</h1>',
```

Comment:
nit: can you make this a string? or add a `merge_variable` called `back_name`?

Reply:
Sure!